pkgs/development/compilers/ghcjs/base.nix

```diff
@@ -159,6 +163,7 @@ in mkDerivation (rec {
         --with-cabal ${cabal-install}/bin/cabal \
         --with-gmp-includes ${gmp.dev}/include \
         --with-gmp-libraries ${gmp.out}/lib
+      ${if isNull ghcLibdir then "" else ''echo "${ghcLibdir}" > "$out/lib/ghcjs-${version}/ghc_libdir"''}
```

Comment:
What is `ghc_libdir`? A quick google search doesn't find anything insightful. Is this something being used specifically by Reflex/Obsidian in some way, or something more general?